scripts/data/wikidata_canadianprovinces.csv

```diff
@@ -0,0 +1,48 @@
+Name,ID
+province of quebec,Q176
+quebec; canada,Q176
+quebec,Q176
+qubec,Q176
+qc,Q176
+quebec province,Q176
+st. john's island,Q1979
+colony of prince edward island,Q1979
+pei,Q1979
+prince edward island,Q1979
+prince edward island; pei,Q1979
+province of prince edward island,Q1979
+ontario,Q1904
+ont,Q1904
+ontario; canada,Q1904
+on,Q1904
+province of alberta,Q1951
+ab,Q1951
+alberta; canada,Q1951
+alberta,Q1951
+manitoba,Q1948
+mb,Q1948
+province of nova scotia,Q1952
+colony of nova scotia,Q1952
+nova scotia,Q1952
+ns,Q1952
+nb,Q1965
+new brunswick,Q1965
+province of new brunswick,Q1965
+colony of new brunswick,Q1965
+british columbia,Q1974
+bc,Q1974
+b.c.,Q1974
+saskatchewan,Q1989
+newfoundland and labrador,Q2003
+newfoundland,Q2003
+nfld,Q2003
+new foundland and labrador,Q2003
+nl,Q2003
+northwest territories,Q2007
+nwt,Q2007
+yukon,Q2009
+yt,Q2009
+yukon territory,Q2009
+the yukon,Q2009
+nunavut,Q2023
+district of keewatin,Q603676
```

Comment:
You can strip the entries for words like "territory" or "province". Basically we don't want entries like "province of new brunswick" instead we just want "brunswick". If you strip away all these qualifiers from the actual province names, then a lot of duplicate entries can also be removed.